PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/com/microsoft/intellij/runner/functionapp/config/FunctionAppConfigurationType.kt

```diff
@@ -24,21 +24,23 @@ package com.microsoft.intellij.runner.functionapp.config
 
 import com.intellij.execution.configurations.ConfigurationType
 import com.microsoft.icons.CommonIcons
+import org.jetbrains.plugins.azure.RiderAzureBundle.message
 import javax.swing.Icon
 
 class FunctionAppConfigurationType : ConfigurationType {
 
     companion object {
         private const val RUN_CONFIG_TYPE_ID = "AzureFunctionAppPublish"
+
+        // TODO: FIX_LOCALIZATION: This name should be moved to RiderAzureMessages
```

Comment:
Leftover TODO?

Reply:
Fixed. Check the API - we can safely move this now.